Formula/gpgme.rb

```diff
@@ -12,7 +13,7 @@ class Gpgme < Formula
     sha256 "4a4c3e90175b02ded974a8a385b6d9799bcc2873c5a1087c558f350eb103934d" => :yosemite
   end
 
-  depends_on "gnupg2"
+  depends_on "gnupg"
```

Comment:
The `inreplace`? Unsure. I added it when we moved from `1.x` to `2.0.x`, partly to ensure it was actually using `2.0.x` and partly to ensure it was always linked to a Homebrew `gpg` rather than interacting with some other one in the environment.

But yes, if the latter isn't a concern and the new `gnupg` 2.1.x ships "vanilla" `gpg` and `gpgsm` executables it should work okay without the `inreplace`.